starter-code/functions-and-arrays.js

```diff
@@ -23,27 +40,44 @@ var words = [
 var longest = findLongestWord(words);
 console.log(longest);
 
+
+
 // Calculating a Sum
 function sumArray (array) {
-
+  var sum = 0;
+  for (i = 0; i < array.length; i++){
+    sum += array[i];
+  }
+  return sum;
```

Comment:
you could try to use the reduce method:

function sumArray (array) {
  return array.reduce(function (sum, number){
    return sum + number;
  }, 0);
}